scripts/osx/README.md

```diff
@@ -1,38 +1,57 @@
-# OSX
+# Building on OSX
+FreeRADIUS can be installed on OSX platforms, however some environment setup is required. Additional dev tools are also configured before the server installation.
 
-Install (Homebrew)[https://brew.sh].
+## Environment Setup
 
-Run the `install_deps.sh` script to install the build dependencies.
-You may want to copy it to a local file and remove things you don't
-need such as postgresql, etc.
+Install the (Homebrew)[https://brew.sh] package manager to streamline the installation process on OSX platforms.
 
-Install Xcode from the (app store)[https://www.apple.com/ca/app-store/].
+Run the `install_deps.sh` script to install the build dependencies. It's recommended to create a copy of this file and edit locally. You can remove apps, libs, or utilites that you don't need such as postgresql. Ensure that you use the updated file when running the script.
+
+Install Xcode from the (app store)[https://www.apple.com/ca/app-store/]. This tool is used to help develop, test, and manage your applications.
 
 Install the xcode command-line tools.
 
 ```
 xcode-select --install
 ```
 
-Update the `~/.zshrc`
+Update the `~/.zshrc` file.
 
 ```
 cat scripts/osx/bash_profile >> ~/.zshrc
 ```
+Note: if using a different shell, ensure you copy the environment paramenters to your current shell.
+
+### Getting the Source
 
-Open a new shell.
+Open a new shell and navigate to the directory where you want to install the FreeRADIUS server.
+
+Download the lastest version from a git repository:
 
 ```
 git clone https://github.com/FreeRADIUS/freeradius-server.git
+```
+
+Alternatively, download the latest version in a zip file:
+
+```
+wget https://github.com/FreeRADIUS/freeradius-server/archive/v4.0.x.zip
```

Comment:
There's no reason to download a zip file.  getting it via `git clone` is the recommended approach.